src/SamplesApp/SamplesApp.Droid/Properties/AndroidManifest.xml

```diff
@@ -2,6 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="uno.platform.unosampleapp" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="19" android:targetSdkVersion="31" />
 	<!-- NOTE: Add permissions in AssemblyInfo.cs -->
```

Comment:
@pkar70 @MartinZikmund Why are permissions added here while this comment mentions they should be in AssemblyInfo?

@jeromelaban This comment was added in https://github.com/unoplatform/uno/pull/3064, do you recall why permissions should be in AssemblyInfo not in AndroidManifest?

Either way, I guess we should just follow the comment and move the permissions.

Reply:
It's generally easier to manipulate these flags in code, as they be placed pretty much anywhere, close to the code using them.